src/PropertyAnnotators/LinksToPropertyAnnotator.php

```diff
@@ -14,15 +14,15 @@
  * @private
  * @ingroup SESP
  *
- * @license GNU GPL v2+
+ * @license GPL-2.0-or-later
  * @since 3.0.4
  */
 class LinksToPropertyAnnotator implements PropertyAnnotator {
 
 	/**
 	 * Predefined property ID
 	 */
-	const PROP_ID = '___LINKSTO';
+	public const PROP_ID = '___LINKSTO';
```

Comment:
This requires PHP 8.1. SMW is keeping compat with 8.0, and this extension still has tests for 7.4, though I suppose those could be removed.

Reply:
The change is in response to `Visibility must be declared on all constants if your | | project supports PHP 7.1 or later`, which seems to be a PHP 7.1 feature, is there a related-change in 8.1 that affects the implementation?

Reply:
Right. I confused 8.1 and 7.1